Exception in thread Thread-166 (_playsoundNix):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/abra/.local/lib/python3.10/site-packages/playsound.py", line 174, in _playsoundNix
    raise PlaysoundException(u'File not found: {}'.format(path))
playsound.PlaysoundException: File not found: /home/abra/Documents/GitHub/ClickSound/system/sounds/1/delete.wav
Exception in thread Thread-301 (_playsoundNix):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/abra/.local/lib/python3.10/site-packages/playsound.py", line 174, in _playsoundNix
    raise PlaysoundException(u'File not found: {}'.format(path))
playsound.PlaysoundException: File not found: /home/abra/Documents/GitHub/ClickSound/system/sounds/1/delete.wav
Exception in thread Thread-468 (_playsoundNix):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/abra/.local/lib/python3.10/site-packages/playsound.py", line 174, in _playsoundNix
    raise PlaysoundException(u'File not found: {}'.format(path))
playsound.PlaysoundException: File not found: /home/abra/Documents/GitHub/ClickSound/system/sounds/1/delete.wav
Exception in thread Thread-518 (_playsoundNix):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/abra/.local/lib/python3.10/site-packages/playsound.py", line 174, in _playsoundNix
    raise PlaysoundException(u'File not found: {}'.format(path))
playsound.PlaysoundException: File not found: /home/abra/Documents/GitHub/ClickSound/system/sounds/1/delete.wav
Exception in thread Thread-695 (_playsoundNix):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/abra/.local/lib/python3.10/site-packages/playsound.py", line 174, in _playsoundNix
    raise PlaysoundException(u'File not found: {}'.format(path))
playsound.PlaysoundException: File not found: /home/abra/Documents/GitHub/ClickSound/system/sounds/1/delete.wav
Exception in thread Thread-899 (_playsoundNix):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/abra/.local/lib/python3.10/site-packages/playsound.py", line 174, in _playsoundNix
    raise PlaysoundException(u'File not found: {}'.format(path))
playsound.PlaysoundException: File not found: /home/abra/Documents/GitHub/ClickSound/system/sounds/1/delete.wav
